Mathlib/Analysis/Calculus/FormalMultilinearSeries.lean

```diff
@@ -302,7 +302,7 @@ noncomputable def fslope (p : FormalMultilinearSeries 𝕜 𝕜 E) : FormalMulti
 theorem coeff_fslope : p.fslope.coeff n = p.coeff (n + 1) := by
   simp only [fslope, coeff, ContinuousMultilinearMap.curryLeft_apply]
   congr 1
-  exact Fin.cons_self_tail 1
+  exact Fin.cons_self_tail (fun _ => (1 : 𝕜))
```

Comment:
Why is this change needed? I certainly hope that the generalisation of Type to Sort doesn't stop me from writing `1` for the constantly one function!

Reply:
Here is the error after changing `Type` to `Sort`:
```lean4
numerals are data in Lean, but the expected type is universe polymorphic and may be a proposition
  (i : Fin (?m.430908 + 1)) → ?m.430909 i : Sort (imax 1 ?u.430907)
```
Perhaps others could weigh in on whether my changes are really necessary, or maybe I should revert some (all) of them?

@eric-wieser also had some concerns in [this thread](https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/.60Fin.2Etake.60.3F/near/472897132)